Cargo.toml

```diff
@@ -1,29 +1,44 @@
 [package]
-name = "rodio"
-version = "0.14.0"
 authors = ["Pierre Krieger <pierre.krieger1708@gmail.com>"]
-license = "MIT OR Apache-2.0"
 description = "Audio playback library"
-keywords = ["audio", "playback", "gamedev"]
-repository = "https://github.com/RustAudio/rodio"
 documentation = "http://docs.rs/rodio"
 edition = "2018"
+keywords = ["audio", "playback", "gamedev"]
+license = "MIT OR Apache-2.0"
+name = "rodio"
```

Comment:
Why do you reorder this? The name is the most important thing. Then come the other things, but in that order please.

Reply:
One of my VS Code extensions has a TOML formatter and I think it tries to order things alphabetically. I'll revert it.